sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs

```diff
@@ -2360,7 +2360,7 @@ public partial class LoadContent
     public partial class ManagedCertificateProperties : Azure.ResourceManager.Cdn.Models.SecretProperties
     {
         public ManagedCertificateProperties() { }
-        public string ExpirationDate { get { throw null; } }
+        public System.DateTimeOffset? ExpirationOn { get { throw null; } }
```

Comment:
The name change of this property is introduced by hard-coded logic in our generator?
Should this be `ExpiredOn`? Do we have similar cases in other RPs?

Reply:
I think so, the logic is [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Mgmt/Decorator/Transformer/RenameTimeToOn.cs). We should have the similar cases in other RPs, for example, we renamed `updatedTime` in ResourceManager to `updatedOn`.

Reply:
Therefore I believe from grammar's perspective, we should have `expiredOn` and `expirationDate`. `expirationOn` does not make much sense.

Reply:
I renamed the property to `ExpiresOn` manually according to the sync meeting discussion, and create an [issue](https://github.com/Azure/autorest.csharp/issues/2316) to make sure we implement it in our generator.